Sources/Model/Receipt/HyperwalletReceipt.swift

```diff
@@ -127,109 +125,204 @@ public struct HyperwalletReceipt: Decodable, Equatable {
 
     /// The transaction type.
     public enum HyperwalletReceiptType: String, Decodable {
-        // Generic fee types
+        /// Generic fee types
```

Comment:
No need to add this in document so just // are fine and remove the extra ///

Reply:
Removed extra /